sdcm/mgmt/cli.py

```diff
@@ -1275,6 +1285,12 @@ def version(self):
         cmd = "version"
         return self.run(cmd=cmd, is_verify_errorless_result=True)
 
+    def feature(self, command: str, feature: str) -> bool:
+        return "--{}".format(feature) in self.run(
```

Comment:
can you supply an example of this command output

Reply:
Added